deploy/overlays/dev/kustomization.yaml

```diff
@@ -43,4 +43,11 @@ patchesStrategicMerge:
     metadata:
       name: website-app
     spec:
-      replicas: 6
+      replicas: 4
```

Comment:
Not sure if you meant to include this as part of this pr

Reply:
reverted and will send another PR